pyton_hw.py

```diff
@@ -0,0 +1,47 @@
+import random
+
+
+def play_coin_flip():
+    player_wins = 0
+    player_losses = 0
+
+    while True:
+        player_choice = input("Виберіть Орел (O) або Решка (Р), або натисніть (X) щоб завершити гру: ")
+        player_choice = player_choice.upper()
+
+        if player_choice == 'X':
```

Comment:
You should include the check for 'x' in while below.
check the following scenario:
- press K
- then press x
- you'll be still the the second while loop as you expect only O or P